WrocRide/Services/UserService.cs

```diff
@@ -0,0 +1,90 @@
+﻿using Microsoft.AspNetCore.Identity;
+using WrocRide.Entities;
+using WrocRide.Exceptions;
+using WrocRide.Models;
+
+namespace WrocRide.Services
+{
+    public interface IUserService
+    {
+        UserDto GetUser();
+        void UpdateUser(UpdateUserDto dto);
+    }
+    public class UserService : IUserService
+    {
+        private readonly WrocRideDbContext _dbContext;
+        private readonly IUserContextService _userContextService;
+        private readonly IPasswordHasher<User> _passwordHasher;
+        public UserService(WrocRideDbContext dbContext, IUserContextService userContextService, IPasswordHasher<User> passwordHasher)
+        {
+            _dbContext = dbContext;
+            _userContextService = userContextService;
+            _passwordHasher = passwordHasher;
+        }
+
+        public UserDto GetUser()
+        {
+            int? userId = _userContextService.GetUserId;
+
+            if (userId == null)
+            {
+                throw new NotFoundException("User not found"); // Change it to NotLoggedIn error
```

Comment:
Tu do usuniecia komentarz + jak nalozysz atrybut autoryzacji na kontroler to ten if jest nie potrzebny, ale nie trzeba go w sumie usuwac bo to dodatkowa weryfikacja.